public-ui/dispatch.yaml

```diff
@@ -0,0 +1,7 @@
+dispatch:
+  - url: "aou-db-test.appspot.com/v1/*"
```

Comment:
This looks fine as long as higher environments can ignore this. I guess just double-check after the next staging push.